client/components/__test__/App.test.js

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
 import { shallow, mount, render } from 'enzyme';
 import toJson from 'enzyme-to-json';
-import App from '../App.jsx';
+import App from '../App';
+import sampleData from '../../../database/sampleData';
 
 describe('App', () => {
-  it('rendsers without crashing', () => {
+  beforeEach(() => {
+    global.fetch = jest.fn(() => Promise.resolve(sampleData));
+  });
+
+  it('renders without crashing', () => {
     const wrapper = shallow(<App />);
     expect(toJson(wrapper)).toMatchSnapshot();
   });
+
+  it('has a function called scoll that increments and decrements the currentPhoto property in state', () => {
+    const wrapper = shallow(<App />);
```

Comment:
Is it typical to include all of these expects in one 'it' block or should they be broken up?